rules/windows/builtin/security/win_security_account_backdoor_dcsync_rights.yml

```
@@ -15,6 +15,7 @@ tags:
 logsource:
     product: windows
     service: security
+    definition: The "Audit Directory Service Changes" logging policy must be configured in order to recieve events. Audit events are generated only for objects with configured system access control lists (SACLs), and only when they are accessed in a manner that matches their SACL settings. This policy covers the following events ids - 5136, 5137, 5138, 5139, 5141
```

```suggestion
    definition: The "Audit Directory Service Changes" logging policy must be configured in order to receive events. Audit events are generated only for objects with configured system access control lists (SACLs) and only when accessed in a manner that matches their SACL settings. This policy covers the following events ids: 5136, 5137, 5138, 5139, 5141
```
Just a typo on "receive", but left a suggestion in the wording